com.gitlab.zehkira.Myuzi.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.gitlab.zehkira.Myuzi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: myuzi
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+modules:
+  - python3-modules.json
+  - name: myuzi
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.gitlab.zehkira.Myuzi.metainfo.xml -t /app/share/metainfo
+      - make prefix=/app install-flatpak
```

Comment:
this is not what I was suggesting to change upstream.

1. you don't need any specific flatpak target. the `install` target should be enough. You should use the exact same naming for the .desktop file, appdata and icons in EVERY case.
2. you still misuse DESTDIR and hardcode a prefix.